lib/AtomsIOPython/src/ase.jl

```diff
@@ -11,26 +11,48 @@ Supported formats:
   - ASE trajectory files
   - [XYZ](https://openbabel.org/wiki/XYZ) and [extxyz](https://github.com/libAtoms/extxyz#extended-xyz-specification-and-parsing-tools) files
 """
-struct AseParser <: AbstractParser end
+@kwdef struct AseParser <: AbstractParser 
+    read::Bool = true
+    guess::Bool = true
+end
 
 
-function AtomsIO.supports_parsing(::AseParser, file; save, trajectory)
+function AtomsIO.supports_parsing(parser::AseParser, file; save, trajectory)
     format = ""
     try
-        format = ase.io.formats.filetype(file; read=!save, guess=false)
+        format = ase.io.formats.filetype(file; read=parser.read, guess=parser.guess)
```

Comment:
I don't think this is correct by the way, since `filetype` will try to open the file if `read=true`. If the file does not exist (because you want to save it) this will not work, will it?

Reply:
Yeah this is true. Due to this the tests are not passing currently. I had not thought about this earlier.

Reply:
That's why we have tests :smile: